tutorial/tutorial-gabc-01.html

```diff
@@ -63,11 +63,11 @@
 
 <p>This tutorial demonstrates the features of Gregorio by showing how to reproduce an existing score.</p>
 
-<p>Documentation for the gabc notation which we use in this tutorial can be found <a href="/gabc/index.html">here</a>. A <a href="/gabc/summary-gabc.pdf">one-page summary is also on-line</a> (PDF).</p>
+<p>Documentation for the gabc notation which we use in this tutorial can be found <a href="../gabc/index.html">here</a>. A <a href="../gabc/summary-gabc.pdf">one-page summary is also on-line</a> (PDF).</p>
```

Comment:
The one-page summary PDF, while still accurate, is pretty outdated.  Should it be updated at this time?